site/content/docs/user/quick-start.md

```diff
@@ -218,13 +218,18 @@ context name `kind` and delete that cluster.
 
 ## Loading an Image Into Your Cluster
 
-Docker images can be loaded into your cluster nodes with:
+You can load an image into your cluster with:
+
+```bash
+kind load docker-image my-app:latest --name test-cluster
+```
 
-`kind load docker-image my-custom-image-0 my-custom-image-1`
+You can also load more than one image at once by passing multiple image names:
+
+```bash
+kind load docker-image my-app:latest my-db:latest my-cache:latest --name test-cluster
+```
 
-> **Note**: If using a named cluster you will need to specify the name of the
-> cluster you wish to load the images into:
-> `kind load docker-image my-custom-image-0 my-custom-image-1 --name kind-2`
```

Comment:
Sorry, please keep the need for passing the cluster name separate as for a very large number of users, they only start the default kind cluster and don't need to worry about additional parameters. That will also keep the examples above more concise.